app/test_split_integer.py

```diff
@@ -2,20 +2,28 @@
 
 
```

Comment:
Add an error message for each test . It's good when the user understands what the problem is